doc/developers/contributing.rst

```diff
@@ -14,6 +14,26 @@ adding new algorithms, and the best way to contribute and to help the project
 is to start working on known issues.
 See :ref:`easy_issues` to get started.
 
+.. topic:: **Our community, our values**
+
+    We are a community based on openness and friendly, didactic,
+    discussions.
+
+    We aspire to treat everybody equally, and value their contributions.
+
+    Decisions are made based on technical merit and consensus.
```

Comment:
> Don't the above three lines go without saying?

I used to think so, and I would expect it. That said, not every community
are actually behaving that way (I don't want to point fingers, but maybe
the most famous open source software...).

We've had several suggestions that it might be useful to be explicit
about these values.